src/main/java/com/epam/training/gen/ai/GenAiTrainingApplication.java

```diff
@@ -4,12 +4,15 @@
 import org.springframework.boot.autoconfigure.SpringBootApplication;
 import org.springframework.context.annotation.PropertySource;
 
+/**
+ * SpringBoot Application main class of the Gen-AI training project.
+ */
 @SpringBootApplication
 @PropertySource("classpath:/config/application.properties")
```

Comment:
You could simply move application.properties file to the root of resources folder, as a result this line could be removed